webpack/scenes/ContentViews/Details/Versions/ContentViewVersionContent.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Text,
+  TextContent,
+  TextVariants,
+} from '@patternfly/react-core';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { urlBuilder } from 'foremanReact/common/urlHelpers';
+
+const ContentViewVersionContent = ({ cvId, versionId, cvVersion }) => {
+  const {
+    deb_count: debCount,
+    docker_manifest_count: dockerManifestCount,
+    docker_tag_count: dockerTagCount,
+    file_count: fileCount,
+    module_stream_count: moduleStreamCount,
+  } = cvVersion;
+
+  return (
+    <React.Fragment>
+      {(moduleStreamCount > 0 &&
+      <React.Fragment id="module_stream_count"><a href={urlBuilder(`content_views/${cvId}/versions/${versionId}/module_streams`, '')}>{`${moduleStreamCount} Module Streams`}</a><br /></React.Fragment>) ||
+        ''
```

Comment:
I don't think you need any of the `|| ''`'s.  JSX will render any `null` or `undefined` just fine without it.

(also, since we're definitely rendering stuff here (`<React.Fragment>`s), this particular one would be moot anyway.)